Okta/okta-system-logs/ingest/parser.yml

```
@@ -134,6 +134,8 @@ stages:
           okta.system.authentication_context.credential.type: '{{parsed_event.message.authenticationContext.credentialType}}'
           okta.system.authentication_context.issuer.id: '{{parsed_event.message.authenticationContext.issuer.id}}'
           okta.system.authentication_context.issuer.type: '{{parsed_event.message.authenticationContext.issuer.type}}'
+          okta.system.outcome.reason: '{{parsed_event.message.outcome.reason}}'
```

I suggest using [`event.code`](https://www.elastic.co/guide/en/ecs/8.4/ecs-event.html#field-event-code) for this purpose:
```suggestion
          event.code: '{{parsed_event.message.outcome.reason}}'
```